tests/base_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+class BaseTest:
+
+    def login_new_user(self, mozwebqa, user):
```

Comment:
I thought about implementing it as a fixture, but I thought that some time ago you expressed dissatisfaction with the way we were doing that in some other suites - fixtures that did things like logging in, as opposed to just providing data. Maybe I misunderstood or misinterpreted, but I assumed you would not like to see the login process as a fixture.